src/bootstrap/llvm.rs

```diff
@@ -568,8 +568,6 @@ fn configure_cmake(
             cfg.define("CMAKE_SYSTEM_NAME", "Haiku");
         } else if target.contains("solaris") || target.contains("illumos") {
             cfg.define("CMAKE_SYSTEM_NAME", "SunOS");
-        } else if target.contains("linux") {
-            cfg.define("CMAKE_SYSTEM_NAME", "Linux");
```

Comment:
There still few places where code removed, is it intended?

Reply:
No, it isn't intended. I just ran `git rebase` but the native content wasn't modified.

Reply:
Reverted the changes that I didn't realized